src/CONST.ts

```diff
@@ -523,6 +523,7 @@ const CONST = {
                 TASKCOMPLETED: 'TASKCOMPLETED',
                 TASKEDITED: 'TASKEDITED',
                 TASKREOPENED: 'TASKREOPENED',
+                MENTIONWHISPER: 'ACTIONABLEMENTIONWHISPER',
```

Comment:
let's call this `ACTIONABLEMENTIONWHISPER`
```suggestion
                ACTIONABLEMENTIONWHISPER: 'ACTIONABLEMENTIONWHISPER',
```